renga/cli/notebooks.py

```diff
@@ -68,10 +70,12 @@ def is_notebook(context):
 
 @notebooks.command()
 @click.option('--engine', default='docker')
+@click.option('--bucket', type=int, envvar='RENGA_BUCKET_ID')
```

Comment:
why use environment variables? not sure it's necessary/useful

we should, however, add an option here to add the code file from disk if `--file` is a path instead of an int